core/domain/wipeout_service.py

```diff
@@ -1399,9 +1516,7 @@ def _pseudonymize_models_transactional(
             if blog_post_summary.author_id == user_id:
                 blog_post_summary.author_id = pseudonymized_id
             blog_post_summary.update_timestamps()
-
-        datastore_services.put_multi(
-            blog_post_models_list + blog_post_summary_models_list)
+        datastore_services.put_multi(blog_post_summary_models_list)
```

Comment:
Same note about not splitting.

Reply:
As mentioned above in the comment https://github.com/oppia/oppia/pull/15827#discussion_r945132048 I have added an Ignore here